docs/src/guide/tutorial.jl

```diff
@@ -0,0 +1,298 @@
+# # Tutorial
+#md # [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/guide/@__NAME__.ipynb)
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/guide/@__NAME__.ipynb)
+
+#nb # DFTK is a Julia package for playing with plane-wave
+#nb # density-functional theory algorithms. In its basic formulation it
+#nb # solves periodic Kohn-Sham equations.
+#
+# This document provides an overview of the structure of the code
+# and how to access basic information about calculations.
+# Basic familiarity with the concepts of plane-wave density functional theory
+# is assumed throughout.
+#
+# For our discussion we will use the classic example of
+# computing the LDA ground state of the silicon crystal.
+# Performing such a calculation roughly proceeds in three steps[^1]:
+#
+# [^1]: Notice that with these parameters the results are still far from convergence in `Ecut` or `kgrid`.
+
+using DFTK
+
+## 1. Define lattice and atomic positions
+a = 10.26  # Silicon lattice constant in Bohr
+lattice = a / 2 * [[0 1 1.];
+                   [1 0 1.];
+                   [1 1 0.]]
+
+## Load HGH pseudopotential for Silicon
+Si = ElementPsp(:Si, psp=load_psp("hgh/lda/Si-q4"))
+
+## Specify type and positions of atoms
+atoms = [Si => [ones(3)/8, -ones(3)/8]]
+
+## 2. Select model and basis
+model = model_LDA(lattice, atoms)
+kgrid = [4, 4, 4]  # k-point grid (Regular Monkhorst-Pack grid)
+Ecut = 7           # kinetic energy cutoff in Hartree
+basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid)
+
+## 3. Run the SCF procedure to obtain the ground state
+scfres = self_consistent_field(basis, tol=1e-8);
+
+# Notice that DFTK by default applies the convergence tolerance `tol`
+# to the energy difference, such that the norm in the density difference
+# is not yet converged to 8 digits.
+#
+# For all quantities DFTK uses
+# [atomic units](https://en.wikipedia.org/wiki/Hartree_atomic_units),
+# where lengths are in Bohr, energies in Hartree and so on.
+# For conversion to other units, DFTK provides conversion factors
+# in the `DFTK.units` module.
+# The convention is that the conversion factors convert *into* atomic units,
+# therefore:
+
+import DFTK.units: eV
+10eV      # 10eV in Hartree
+#-
+1.2 / eV  # 1.2 Hartree in eV
```

Comment:
This is redundant. Remove or use Angstroms?

Reply:
Not really it shows the opposite direction

Reply:
Yeah but it follows directly, we're not teaching multiplication and division here

Reply:
I find unit conversion extremely confusing and believe it or not, realising this helps me a lot.

Reply:
I completely agree that's units are extremely confusing, but it's just not something that belongs to the doc. But I don't feel that strongly about it. Maybe just use angstroms for a change, which is a good opportunity to tell people how to input unicode in a comment.